### ISSUE TITLE
issue: select_node.jstree not triggered on initial ajax

### ISSUE THREAD

**Issue by qahar** [vakata/jstree#1772]
I use jstree v3.3.4 and currently use ajax with proper return the state-selected true on one of the node.
This initial load is same as no 'jstree' on local storage, or you remove them.

When there is 'jstree' in local storage, select_node triggered as expected and I can use it to do certain action to load the folder content when page reloaded.

For the initial load (no jstree local storage), select.node event is not triggered; while changed.jstree trigger 'model' and 'ready' action.

**Request**
Please trigger select.node if node state-selected is true in ajax.

Incase help others, my current sollution is:

```
var ls_tree = JSON.parse(localStorage.getItem('jstree'));

$('#jstree').on('select_node.jstree', function(e, data) {
    // Node selected, do something..
}).on('changed.jstree', function(e, data) {
    if (data.action == 'ready' && $.isEmptyObject(ls_tree)) {
        // Initial page load no previous node selected, do something..
    }
})
```

If any other better solution, I'm all ears..
Thank for the great plugin!

**Comment by vakata** [comment]
To capture changes to selection only the `changed.jstree` event is reliable - `select_node` only fires when the corresponding function is explicitly called. Triggering select_node all the time is quite heavy - it will bring the tree to a stop if using cascading checkboxes for example. 

TLDR: to process selection always use the `changed.jstree` event.

**Comment by qahar** [comment]
thank you for the tips
